boards/boardctl.c

```diff
@@ -452,6 +452,31 @@ int boardctl(unsigned int cmd, uintptr_t arg)
         break;
 #endif
 
+#ifdef CONFIG_BOARDCTL_BOOT_IMAGE
+      /* CMD:           BOARDIOC_BOOT_IMAGE
+       * DESCRIPTION:   Boot a new application image.
+       *                Execute the required actions for booting a new
+       *                application image (e.g. deinitialize peripherals,
```

Comment:
change all application to system?

Reply:
There is an [RFC](https://datatracker.ietf.org/doc/html/rfc9019) in the works for standardizing the firmware update process for IoT.
They are using **Application Firmware Image** to refer to the firmware image that is not the bootloader.
https://datatracker.ietf.org/doc/html/rfc9019#section-2.1

I believe we should stick to that. What do you think?

Reply:
Ok